src/models/ili9225.rs

```diff
@@ -0,0 +1,251 @@
+const ILI9225_POWER_CTRL2: u8 = 0x11;
+const ILI9225_POWER_CTRL3: u8 = 0x12;
+const ILI9225_POWER_CTRL4: u8 = 0x13;
+const ILI9225_POWER_CTRL5: u8 = 0x14;
+
+const ILI9225_DRIVER_OUTPUT_CTRL: u8 = 0x01; // Driver Output Control
+const ILI9225_LCD_AC_DRIVING_CTRL: u8 = 0x02; // LCD AC Driving Control
+const ILI9225_ENTRY_MODE: u8 = 0x03; // Entry Mode
+const ILI9225_DISP_CTRL1: u8 = 0x07; // Display Control 1
+const ILI9225_BLANK_PERIOD_CTRL1: u8 = 0x08; // Blank Period Control
+const ILI9225_FRAME_CYCLE_CTRL: u8 = 0x0B; // Frame Cycle Control
+const ILI9225_INTERFACE_CTRL: u8 = 0x0C; // Interface Control
+const ILI9225_OSC_CTRL: u8 = 0x0F; // Osc Control
+const ILI9225_VCI_RECYCLING: u8 = 0x15; // Osc Control
+const ILI9225_RAM_ADDR_SET1: u8 = 0x20; // Osc Control
+const ILI9225_RAM_ADDR_SET2: u8 = 0x21; // Osc Control
+
+const ILI9225_GATE_SCAN_CTRL: u8 = 0x30; // Gate Scan Control Register
+const ILI9225_VERTICAL_SCROLL_CTRL1: u8 = 0x31; // Vertical Scroll Control 1 Register
+const ILI9225_VERTICAL_SCROLL_CTRL2: u8 = 0x32; // Vertical Scroll Control 2 Register
+const ILI9225_VERTICAL_SCROLL_CTRL3: u8 = 0x33; // Vertical Scroll Control 3 Register
+const ILI9225_PARTIAL_DRIVING_POS1: u8 = 0x34; // Partial Driving Position 1 Register
+const ILI9225_PARTIAL_DRIVING_POS2: u8 = 0x35; // Partial Driving Position 2 Register
+const ILI9225_HORIZONTAL_WINDOW_ADDR1: u8 = 0x36; // Horizontal Address Start Position
+const ILI9225_HORIZONTAL_WINDOW_ADDR2: u8 = 0x37; // Horizontal Address End Position
+const ILI9225_VERTICAL_WINDOW_ADDR1: u8 = 0x38; // Vertical Address Start Position
+const ILI9225_VERTICAL_WINDOW_ADDR2: u8 = 0x39; // Vertical Address End Position
+
+const ILI9225_GAMMA_CTRL1: u8 = 0x50; // Gamma Control 1
+const ILI9225_GAMMA_CTRL2: u8 = 0x51; // Gamma Control 2
+const ILI9225_GAMMA_CTRL3: u8 = 0x52; // Gamma Control 3
+const ILI9225_GAMMA_CTRL4: u8 = 0x53; // Gamma Control 4
+const ILI9225_GAMMA_CTRL5: u8 = 0x54; // Gamma Control 5
+const ILI9225_GAMMA_CTRL6: u8 = 0x55; // Gamma Control 6
+const ILI9225_GAMMA_CTRL7: u8 = 0x56; // Gamma Control 7
+const ILI9225_GAMMA_CTRL8: u8 = 0x57; // Gamma Control 8
+const ILI9225_GAMMA_CTRL9: u8 = 0x58; // Gamma Control 9
+const ILI9225_GAMMA_CTRL10: u8 = 0x59; // Gamma Control 10
+
+fn options_write_cmd<DI>(di: &mut DI, options: &ModelOptions) -> Result<(), DI::Error>
+where
+    DI: Interface,
+{
+    let rotation = (options.orientation.rotation as u8) % 4; // Only accept 0-3
+
+    // Command 1: DRIVER_OUTPUT_CTRL (0x01)
+    let driver_high_byte = match rotation {
+        0 => 0x01, // 0°
+        1 => 0x00, // 90°
+        2 => 0x02, // 180°
+        3 => 0x03, // 270°
+        _ => 0x01, // Not reachable
+    };
+    let driver_params = [driver_high_byte, 0x1C];
+    di.write_raw(ILI9225_DRIVER_OUTPUT_CTRL, &driver_params)?;
+
+    // Command 2: ENTRY_MODE (0x03)
+    let color_order_byte = if options.color_order == ColorOrder::Bgr {
+        0x10
+    } else {
+        0x00
+    };
```

Comment:
Writing this using a match statement is a bit shorter and safer if we ever decide to add another color order.
```suggestion
    let color_order_byte = match options.color_order {
        ColorOrder::Rgb => 0x00,
        ColorOrder::Bgr => 0x10,
    };
```